build/scripts/cibuild.ps1

```diff
@@ -45,6 +45,12 @@ function Terminate-BuildProcesses() {
 
 try {
     . (Join-Path $PSScriptRoot "build-utils.ps1")
+
+    if ($badArgs -ne $null) {
```

Comment:
❗️ I believe this needs to come after the `Push-Location` line, since the `finally` will execute.